sql/moz-fx-data-shared-prod/firefox_ios_derived/firefox_ios_clients_v1/init.sql

```diff
@@ -0,0 +1,172 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.firefox_ios_derived.firefox_ios_clients_v1`
+AS
+-- Query first observations for Firefox iOS Clients.
+WITH first_seen AS (
+  SELECT
+    client_id,
+    submission_date,
+    first_seen_date,
+    sample_id,
+    country AS first_reported_country,
+    isp AS first_reported_isp,
+    normalized_channel AS channel,
+    device_manufacturer,
+    device_model,
+    normalized_os_version AS os_version,
+    app_display_version AS app_version
+  FROM
+    firefox_ios.baseline_clients_first_seen
+  WHERE
+    submission_date < CURRENT_DATE
+    AND client_id IS NOT NULL
+),
+-- Find the most recent activation record per client_id.
+activations AS (
+  SELECT
+    client_id,
+    is_activated,
+  FROM
+    firefox_ios_derived.new_profile_activation_v2
```

Comment:
Query console says you need a partition filter here

Reply:
yes, thank you. Fixed that.